eng/vmr-validation.yml

```diff
@@ -0,0 +1,57 @@
+### This pipeline validates new changes of the VMR (https://github.com/dotnet/dotnet)
+### VMR is cloned and updates are applied locally
+
+parameters:
+- name: vmrBranch
+  displayName: dotnet/dotnet branch to use
+  type: string
+  default: $(System.PullRequest.TargetBranch)
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - main
+
+resources:
+  repositories:
+  - repository: vmr
+    type: github
+    name: dotnet/dotnet
+    endpoint: public
+
+variables:
+- template: /eng/common/templates/variables/pool-providers.yml
+
+- name: vmrPath
+  value: $(Agent.BuildDirectory)/vmr
+
+- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+  - group: DotNetBot-GitHub
+- ${{ else }}:
+  - name: BotAccount-dotnet-bot-repo-PAT
+    value: N/A
+
+jobs:
```

Comment:
Consider defining a job template for VMR synchronization that is shared between vmr-synchronization and vmr-validation - they appear identical except for vmr-synchronization has one additional step that could be conditioned or passed as a steps parameter.

Reply:
They are very similar at the moment but that is because I took the WIP I've sent you and dumbed it down to get this simple PR check in right away to have something running. I plan to bring those changes back soon and they will change this pipeline dramatically so I won't de-duplicate them now.

I will see what will be left as common after.

Reply:
Fair enough.